templates/entry/_options.html.twig

```diff
@@ -1,136 +1,227 @@
+                       class="{{ html_classes({'active': criteria.getOption('time') == '3h' }) }}">
                         {{ '3h'|trans }}
                     </a></li>
                 <li>
                     <a href="{{ options_url('time', '6h') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '6h')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '6h' }) }}">
                         {{ '6h'|trans }}
                     </a>
                 </li>
                 <li>
                     <a href="{{ options_url('time', '12h') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '12h')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '12h' }) }}">
                         {{ '12h'|trans }}
                     </a>
                 </li>
                 <li>
                     <a href="{{ options_url('time', '1d') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '1d')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '1d' }) }}">
                         {{ '1d'|trans }}
                     </a>
                 </li>
                 <li>
                     <a href="{{ options_url('time', '1w') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '1w')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '1w' }) }}">
                         {{ '1w'|trans }}
                     </a>
                 </li>
                 <li>
                     <a href="{{ options_url('time', '1m') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '1m')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '1m' }) }}">
                         {{ '1m'|trans }}
                     </a>
                 </li>
                 <li>
                     <a href="{{ options_url('time', '1y') }}"
-                       class="{{ html_classes({'active': route_has_param('time', '1y')}) }}">
+                       class="{{ html_classes({'active': criteria.getOption('time') == '1y' }) }}">
                         {{ '1y'|trans }}
                     </a>
                 </li>
             </ul>
         </li>
         <li class="dropdown">
-            <button
-                    aria-label="{{ 'filter_by_type'|trans }}"
-                    title="{{ 'filter_by_type'|trans }}"><i
-                        class="fa-solid fa-filter" aria-hidden="true"></i>
+            <button aria-label="{{ 'filter_by_type'|trans }}"
+                    title="{{ 'filter_by_type'|trans }}">
+                <i aria-hidden="true" class="
+                    {% if criteria.getOption('type') == 'all' %} fa-solid fa-file
```

Comment:
I now this would be longer, but for better readability I'd go with
```twig
{% if ... %}
    statement
{% elseif ... %}
    ...
{% endif %}
```

Reply:
Done